pojo.go

```diff
@@ -210,6 +210,46 @@ func RegisterPOJO(o POJO) int {
 	return structInfo.index
 }
 
+// easy for test
+func UnRegisterPOJOs(os ...POJO) []int {
+	arr := make([]int, len(os))
+	for i := range os {
+		arr[i] = UnRegisterPOJO(os[i])
+	}
+
+	return arr
+}
+
+func UnRegisterPOJO(o POJO) int {
+	pojoRegistry.Lock()
+	defer pojoRegistry.Unlock()
+
+	var structInfo structInfo
+	v := reflect.ValueOf(o)
+	switch v.Kind() {
+	case reflect.Struct:
+		structInfo.typ = v.Type()
+	case reflect.Ptr:
+		structInfo.typ = v.Elem().Type()
+	default:
+		structInfo.typ = reflect.TypeOf(o)
+	}
```

Comment:
these lines can be extracted to a new func

Reply:
I will fix it.